app/Models/Poll.php

```diff
@@ -120,9 +120,9 @@ public function setTitleAttribute($title)
     public function makeSlugFromTitle($title)
     {
         $slug = strlen($title) > 20 ? substr(Str::slug($title), 0, 20) : Str::slug($title);
-        $count = $this->where('slug', 'LIKE', "%$slug%")->count();
+        $count = $this->where('slug', 'LIKE', sprintf('%%s%', $slug))->count();
```

Comment:
This might be okay, but it looks like the type of thing that we should verify. I'd be a bit surprised if additional escaping isn't necessary.

Reply:
Here's an example of why I say this, in Tinker:

```php
>>> $slug = 'foo';
=> "foo"
>>> sprintf('%%s%', $slug);
=> "%s"
```

And without the additonal `%s` instances:

```php
>>> sprintf('%s', $slug);
=> "foo"
```

Reply:
I actually overlooked this and prefer how was from the start. I dont believe sprintf() should be used like this in a where statement. Agree?

Reply:
Agreed; I think it was fine to begin with.